src/github/client/installation-token-cache.ts

```diff
@@ -0,0 +1,43 @@
+import LRUCache from "lru-cache";
+import AuthToken from "./auth-token";
+
+/**
+ * A cache that holds installation tokens for the most recently used installations.
+ *
+ * An installation token can be used to get data from a GitHub org in which the app has been installed (i.e. get commits, pull requests, etc.).
+ *
+ * @see https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-an-installation
+ */
+export default class InstallationTokenCache {
+	private readonly installationTokenCache: LRUCache<number, AuthToken>;
```

Comment:
That's right. I was thinking about the same. Does Octokit use Redis for tokens storage? If not, we'll probably by fine with the rollout

Reply:
The move to SQS should have nothing to do with the cache. We'll still need the cache then. 

It shouldn't be hard to put this cache into Redis, but that's an improvement, not a show stopper, so I'd like to move that into a follow up task.

Reply:
Ah, I get what you mean by " only using Redis for this cache" now. This would be the only use case for Redis and Redis would be overkill for that. Yeah, maybe. We should start with this per-process cache and see how it goes and then go from there.

Reply:
We use Redis for multiple things, now for deduplicating backfill queue, so we'll still need it even after migrating to SQS.

Reply:
All of which could be moved to the DB fairly easily.  The costs (monetary/maintenance/time) isn't really worth it IMO unless we are doing _a lot_ of requests and are trying to offload the DB